api/nodes/serializers.py

```diff
@@ -492,7 +492,10 @@ def update(self, instance, validated_data):
                 raise exceptions.NotFound
             if not inst.auth(user):
                 raise exceptions.PermissionDenied
```

Comment:
Discussed in person: `PermissionsError` might not be appropriate; this is more of a validation rather than a permissions error, so the user should receive a 400 rather than a 403. You can use DRF's ValidationError for this.